client/src/components/DataFiles/DataFilesListing/DataFilesListingCells.jsx

```diff
@@ -55,10 +55,11 @@ export const FileNavCell = React.memo(
     const previewCallback = (e) => {
       e.stopPropagation();
       e.preventDefault();
-      if (api === 'googledrive') {
-        window.open(href, '_blank');
-        return;
-      }
+      // Temporary Integrations Filtering and conditional rendering of Google Drive: WP-24
+      // if (api === 'googledrive') {
+      //   window.open(href, '_blank');
+      //   return;
+      // }
```

Comment:
```suggestion
```
You can go ahead and delete this code instead of commenting it out. Git history will save the changes and let us know what they were.